include/ppx/geometry.h

```diff
@@ -102,6 +104,11 @@ struct GeometryOptions
     GeometryOptions& AddTangent(grfx::Format format = grfx::FORMAT_R32G32B32A32_FLOAT);
     GeometryOptions& AddBitangent(grfx::Format format = grfx::FORMAT_R32G32B32_FLOAT);
 
+    // NOTE: Setting max index/vertex counts will set up index/vertex
+    // buffers for "Use #2"
```

Comment:
It's kind of unclear of what "Use #2" means here, as fas as I understand it refers to line 148-151. But i think it is better to be explicit.